### DISCUSSION TITLE
Forced Combination wont obey!

### DISCUSSION THREAD

**Discussion by BakeSloth** [nftchef/art-engine#179] [Help + Q&A]
Short story - 
Is there a way I can weigh forcedCombinations?
Any way of setting the priority of one forcedcombo in relation to others?

Long story - 
So I have two forced combination rules set.
one that forces background color folder to Outfit folder and Helmet folder.
The other forces trait filename to filename.

But It seems I can't override the first existing forcedCombination.

  BGRed: ["OutfitRed", "HelmetRed"],
  "Wings Red": ["Halo Red"],

My Folder structure and layer order is : (i will only list the Red folders, but I have folders for all primary colors named similarly)
-Background
--BGRed
---Red Solid.png
---Red Gradient.png
-Outfit
--OutfitRed
---Armor Red.png
---Shirt Red.png
---Wings Red.png
-Helmet
--HelmetRed
---Spartan Red.png
---Crown Red.png
---Halo Red.png

If a Red background is first selected, The outfit and Helmet are chosen from their Red folder as expected!
But I cant also seem to force a specific Red Outfit to match a specific Red Helmet trait. 
It just defaults to the first rule matching background colors to outfit and helmet, even though the next rule doesn't logically conflict in any way.

Any ideas on how I can get this to work?

I basically just want to match filename to filename, while still keeping the working rule that precedes it.

**Comment by cryptofannatic** [comment]
did you manage to figure this out? 
Also, I noticed that you're not using any weights on your folders or filenames. Maybe that could be the issue?
